ydb/docs/en/core/reference/configuration/spilling.md

```diff
@@ -0,0 +1,256 @@
+- The directory must have sufficient write and read permissions for the user under which ydbd is running
+
+**Recommendations:**
+
+- Use a separate disk or partition for spilling
+- Preferably use fast storage devices (SSD/NVMe)
+- Ensure sufficient free space is available
+
+**Possible errors:**
+
+- **Permission denied** — insufficient directory access permissions. See [Spilling Troubleshooting](../../troubleshooting/spilling.md#permission-denied)
+
+#### MaxTotalSize
+
+**Type:** `uint64`  
+**Default:** `21474836480` (20 GiB)  
+**Description:** Maximum total size of all spilling files. When the limit is exceeded, spilling operations fail with an error.
+
+**Recommendations:**
+
+- Set the value based on available disk space
+
+**Possible errors:**
+
+- **Total size limit exceeded: X/YMb** — maximum total size of spilling files exceeded. See [Spilling Troubleshooting](../../troubleshooting/spilling.md#total-size-limit-exceeded)
+
+### Thread Pool Configuration (TIoThreadPoolConfig)
+
+{% note info %}
+
+I/O pool threads for spilling are created in addition to threads allocated for the [actor system](../../concepts/glossary.md#actor-system). When planning the number of threads, consider the overall system load.
+
+{% endnote %}
+
+#### WorkersCount
+
+**Type:** `uint32`  
+**Default:** `2`  
+**Description:** Number of worker threads for processing spilling I/O operations.
+
+**Recommendations:**
+
+- Increase for high-load systems
+- Consider the number of CPU cores on the server
+
+**Possible errors:**
+
+- **Can not run operation** — I/O thread pool operation queue overflow. See [Spilling Troubleshooting](../../troubleshooting/spilling.md#can-not-run-operation)
+
+#### QueueSize
+
+**Type:** `uint32`  
+**Default:** `1000`  
+**Description:** Size of the spilling operations queue. Each task sends only one data block to spilling at a time, so large values are usually not required.
+
+**Possible errors:**
+
+- **Can not run operation** — I/O thread pool operation queue overflow. See [Spilling Troubleshooting](../../troubleshooting/spilling.md#can-not-run-operation)
+
+## Memory Management
+
+### Relationship with memory_controller_config
+
+Spilling activation is closely related to memory controller settings. Detailed `memory_controller_config` configuration is described in a [separate article](../../reference/configuration/index.html#memory-controller).
+
+The key parameter for spilling is **`activities_limit_percent`**, which determines the amount of memory allocated for query processing activities. This parameter affects the available memory for user queries and, accordingly, the frequency of spilling activation.
+
+**Impact on spilling:**
+
+- When increasing `activities_limit_percent`, less memory remains for query execution → spilling activates more frequently
+- When decreasing `activities_limit_percent`, more memory is available for queries → spilling activates less frequently
```

Comment:
Let's update the memory controller documentation with content like this. IIRC, there was nothing about spilling in the initial version.

Mention this interaction in `concepts/spilling.md` too.